aten/src/ATen/native/cpu/PowKernel.cpp

```diff
@@ -148,89 +148,13 @@ void pow_tensor_scalar_kernel(TensorIterator& iter, Scalar exp_scalar) {
       }
     });
   } else {
-    // Integral types do not allow AVX2 vector optimizations for pow/sqrt/rsqrt.
-    // Trying to implement pow/sqrt/rsqrt as loop in vec256_int.h does not allow
-    // powering integral tensor to float exponent. That's why we need this code
-    // duplication:
```

Comment:
@pbelevich with type promotion the base and exp should get promoted to the same type (both scalar_t) so integral tensor to float exponent doesn't make sense to me here. I had some build failures in avx that I'm trying to fix though so maybe I'm misunderstanding something here?